README.md

```diff
@@ -60,7 +60,7 @@ Current properties include:
 - **operations:** List of implemented operations. Use the IDs of the operations in this [standard](http://standards.ncats.io/operation.json).
 - **batch_size_limit:** Maximum number of CURIEs allowed in _any_ 'ids', 'categories', or 'predicates' list. -1 indicates no limit. See ImplementationRules.md in the TRAPI standard github repo for details. 
 - **rate_limit**: Maximum number of requests allowed per _minute_ from each client. -1 indicates no limit.
-- **test_data_location:** URL that resolves to [SRI-Testing-harness](https://github.com/TranslatorSRI/SRI_testing) -compliant test data in some public internet location (e.g. in the API implementation source code repository).
+- **test_data_location:** URL that resolves to [SRI-Testing-harness](https://github.com/TranslatorSRI/SRI_testing) -compliant test data, in a public internet location (e.g. in the API implementation source code repository). Note that the several alternate formats of **test_data_location** property value specification is now support: (legacy) simply single REST URL string; a JSON array of such URL's; a JSON object with one or more JSON dictionary tags chosen from "default" and/or a valid x-maturity drawn from "production", "staging", "testing" or "development" where the dictionary values may be simple URL strings or JSON array (as above). Note: the use of "default" indicates that the given test data may be used for any and all endpoints of `x-maturity` environments, specified in the Translator SmartAPI Registry entry's "servers" block, which are not otherwise explicitly specified here in the **test_data_location** JSON object.
```

Comment:
Perhaps we can revise for grammar / structure, starting from the sentence "Note..."? I give my revision below...

> Note that several formats are now supported:
> * string (a single url; original format)
> * array of such strings
> * object with one or more key-value pairs. 
>   * key options are "default", "production", "staging", "testing", "development" (the last 4 correspond to the x-maturity level of different servers, defined by [Architecture/ITRB](https://github.com/NCATSTranslator/TranslatorArchitecture/blob/master/SmartAPIRegistration.md#environments)
>   * "default" test data may be used to test servers of any x-maturity level. 
>   * value is either a string or array (defined above)